test/interop.spec.js

```diff
@@ -9,30 +9,27 @@ const expect = chai.expect
 chai.use(dirtyChai)
 const dagCBOR = require('../src')
 const loadFixture = require('aegir/fixtures')
-const bs58 = require('bs58')
 const isNode = require('detect-node')
-const CID = require('cids')
 
 const arrayLinkCBOR = loadFixture('test/fixtures/array-link.cbor')
-const arrayLinkJSON = require('./fixtures/array-link.json')
-const arrayLink = arrayLinkJSON.map(x => new CID(x['/']))
+const arrayLinkJS = require('./fixtures/array-link.js')
 
 const emptyArrayCBOR = loadFixture('test/fixtures/empty-array.cbor')
-const emptyArrayJSON = require('./fixtures/empty-array.json')
+const emptyArrayJS = require('./fixtures/empty-array.js')
```

Comment:
I think you should ditch the `JS` in the const names and the `.js` from the end of the required names, let's pretend like they never were JSON and do it as if they were always .js.

Reply:
I guess the `JS` could be justified makes sense given we have `CBOR` as well. `.js` should still go IMO.